README.md

```diff
@@ -1,5 +1,46 @@
 # TQ1 DOCS
 
-This repository contains all general tq1 docs that do not apply to the SDKs
+## What is TQ1?
 
-- master-reports-files : Read the docs [link](http://tq1-reports-files.readthedocs.org/)
+TQ1 a web platform made by Taqtile used to manage mobile-marketing campaigns. By using it you'll be able to create and schedule push notification campaigns, either by sending targeted notifications or by creating geofences that will target users according to location.
+
+It consists of 3 main parts: the SDKs, the admin and the server.
+
+In this docs you will find the links to all our documentation.
+
+To find more details on how to use TQ1 admin and FAQ, click [here](http://help.tq1.taqtile.io/) (you will have to login in order to view the content)
+
+## TQ1 basic flow
+
+The SDKs will send all information needed to the server, while the admin is used to create, manage and view results of  campaigns.
+
+The basic flow can be seen on the following image:
+
+![](./images/TQ1.png)
+
+## TQG basic flow
+
+TQG is Taqtile's geolocation platform. It will be used to detect when user entered the desired location in order to trigger a notification.
+
+Here is TQG basic flow:
+
+![](./images/tqg.png)
+
+## TQ1 + TQG
+
+TQG can be used along with TQ1, so you can manage everything using TQ1 interface. When doing this the flow will be like this:
```

Comment:
also add sth like "This is the default configuration but other use cases for TQG will be shown later on the document"